tests/cargo_test.rs

```diff
@@ -12,37 +13,43 @@ const SNAPSHOT_PATH_C: &str = "../repositories/snapshots/cargo_ci/";
 
 #[test]
 fn test_cargo_binary() {
-    let tmp_dir = temp_dir().join("cargo");
-    let data = TemplateData::new(&tmp_dir).license("MIT").branch("master");
+    let tmp_dir = temp_dir();
+    create_dir_all(tmp_dir.join("cargo")).unwrap();
+    let path = tmp_dir.join("cargo");
+    let data = TemplateData::new(&path).license("MIT").branch("master");
 
     Cargo::new()
         .docker_image_description("description-docker")
         .create_ci(data)
         .unwrap();
-    compare_template_skip(Path::new(SNAPSHOT_PATH_B), &tmp_dir, SKIPPED_FOLDERS);
+    compare_template_skip(Path::new(SNAPSHOT_PATH_B), &path, SKIPPED_FOLDERS);
 }
 
 #[test]
 fn test_cargo_library() {
-    let tmp_dir = temp_dir().join("cargo_library");
-    let data = TemplateData::new(&tmp_dir).license("MIT").branch("main");
+    let tmp_dir = temp_dir();
+    create_dir_all(tmp_dir.join("cargo_library")).unwrap();
+    let path = tmp_dir.join("cargo_library");
+    let data = TemplateData::new(&path).license("MIT").branch("main");
 
     Cargo::new()
         .docker_image_description("description-docker")
         .create_lib()
         .create_ci(data)
         .unwrap();
-    compare_template_skip(Path::new(SNAPSHOT_PATH_L), &tmp_dir, SKIPPED_FOLDERS);
+    compare_template_skip(Path::new(SNAPSHOT_PATH_L), &path, SKIPPED_FOLDERS);
 }
 #[test]
 fn test_cargo_ci() {
-    let tmp_dir = temp_dir().join("cargo_only_ci");
-    let data = TemplateData::new(&tmp_dir).license("MIT").branch("main");
+    let tmp_dir = temp_dir();
+    create_dir_all(tmp_dir.join("cargo_ci")).unwrap();
+    let path = tmp_dir.join("cargo_ci");
```

Comment:
Do we need to change `cargo_only_ci` into `cargo_ci`? Is there a particular reason?

Reply:
If you are referring to the only_ci() function, that has not been replaced. What appears is just the name of the folder containing the temporary tests with only the ci files.